automation/issuer/bot/issuer_bot.py

```diff
@@ -72,7 +72,7 @@ def handle_handle_registered_investor(event):
         commands = []
         for token_cid, token in tokens.items():
             logging.info(f"token_cid: {token_cid}, token: {token}")
-            if client.party in token['id']['signatories']['textMap']:
+            if client.party in token['id']['signatories']['textMap'] and token['isPublic']:
```

Comment:
the decorated function names need changing. It looks like there is also a name collision `handle_handle_registered_investor` is used twice. I am addressing the naming in a PR so we may have a mini conflict there